lib/runner.js

```diff
@@ -46,6 +46,8 @@ util.inherits(Runner, EventEmitter);
 module.exports = Runner;
 
 optionChain(chainableMethods, function (opts, title, fn) {
+	fn = fn || function () {};
```

Comment:
I think having `function noop() {}` at the top of the file and using it here instead of anonymous function inline would be better.